src/cloudant/account.py

```diff
@@ -440,16 +446,70 @@ def cors_origins(self):
         Retrieve a list of CORS origins
 
         """
-        pass
+        cors = self.cors_configuration()
+
+        return cors['origins']
 
     def update_cors_configuration(
             self,
             enable_cors=True,
             allow_credentials=True,
-            *origins):
+            origins=[],
+            overwrite_origins=False):
         """
+        _update_cors_configuration_
 
-        PUT /_api/v2/user/config/cors   Changes the CORS configuration
+        Merge existing CORS config with updated values and write to
+          the API endpoint
+
+        :param boolean enable_cors: Enable/disables cors
+        :param boolean allow_credentials: Allows authentication creds
+        :param list origins: Allowed CORS origin(s)
+            ["*"]: any origin
+            []: disabled/no origin(s)
+        :param boolean overwrite_origins: Default concatinate new 'origins'
+            list to old one, else replace 'origins' with new list
+        """
+        cors_config = {
+            'enable_cors': enable_cors,
+            'allow_credentials': allow_credentials,
+            'origins': origins
+        }
+
+        if overwrite_origins:
+            return self._write_cors_configuration(cors_config)
+
+        old_config = self.cors_configuration()
+
+        # update config values
+        updated_config = old_config.copy()
+
+        updated_config['enable_cors'] = cors_config.get('enable_cors')
+        updated_config['allow_credentials'] = cors_config.get('allow_credenials')
+
+        if cors_config.get('origins') == ["*"]:
+            updated_config['origins'] = ["*"]
+        elif old_config.get('origins') != cors_config.get('origins'):
```

Comment:
You may want to do something like make these both sets and add them to remove duplicates